eva/utils/kv_cache.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pickle
+import shutil
+from typing import Any
+
+from diskcache import FanoutCache
```

Comment:
Just curious about the choice of this library. Are we trying to optimize writes to disk here?

Reply:
I was looking for a lightweight caching library, ideally with a low setup cost. `FanoutCache` optimizes for writes.